README.md

```
@@ -52,6 +56,14 @@ To build an executable containing a single library's unit tests:
 task build:unit-test-<lib_name>
 ```
 
+<h2 id="installing-header">Installing</h2>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: h2

(MD033, no-inline-html)